client/src/app/site/agenda/components/item-info-dialog/item-info-dialog.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material';
+import { ViewItem } from '../../models/view-item';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { itemVisibilityChoices } from 'app/shared/models/agenda/item';
```

Comment:
This import should use relative paths, like the `ViewItem`.

Additionally these imports should be sorted.

Reply:
the relative path to item.ts is pretty long, therefore I prefer the absolute